src/github.com/matrix-org/dendrite/clientapi/readers/profile.go

```diff
@@ -38,6 +43,12 @@ type displayName struct {
 	DisplayName string `json:"displayname"`
 }
 
+type profileUpdate struct {
```

Comment:
This should really be somewhere in common that other packages can import.
That way the components that consume the topic can use the same structure when parsing incoming messages.
I'm not certain where we want to put that sort of thing in dendrite yet, so there's no need to do so in this PR.
But it'd be good to have a TODO marker here so that we don't forget.